starters/solidstart-tanstackquery-tailwind-modules/src/routes/index.tsx

```diff
@@ -4,7 +4,7 @@ export default function Home() {
   return (
     <>
       <header class="flex justify-center items-center text-white my-5 mx-auto bg-blue-500  w-full lg:w-[75%] p-4 text-lg ">
-        Solid Start - Tanstack Query - Tailwind CSS - CSS Modules Kit
+        Solid Start
```

Comment:
Same as above.

I am not sure he meant for you to remove other technology stacks